src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/CourtEventCharge.kt

```diff
@@ -63,7 +63,7 @@ class CourtEventCharge(
 
   val propertyValue: BigDecimal?,
 
-  val totalPropertyValue: BigDecimal?,
+  val totalPropertyValue: BigDecimal?, // offences taken into consideration button on OCDCCASE
```

Comment:
is that copy n paste comment from above?

Reply:
No it's the same pop up